catroid/src/main/java/org/catrobat/catroid/bluetooth/ConnectBluetoothDeviceActivity.java

```diff
@@ -117,12 +124,14 @@ private String getSelectedBluetoothAddress(View view) {
 
 		@Override
 		public void onItemClick(AdapterView<?> av, View view, int position, long id) {
-
 			String address = getSelectedBluetoothAddress(view);
-			if (address == null) {
+			Pair pair = newDevicesArrayAdapter.getItem(position);
+			if (address == null || pair == null) {
 				return;
 			}
-			connectDevice(address);
+			if (pair.second.equals(DEVICE_TYPE_CLASSIC)) {
+				connectDevice(address);
+			}
```

Comment:
Since the logic for connecting is done in another ticket, this seems to be fine :+1: